evap/grades/forms.py

```diff
@@ -10,7 +10,6 @@ class GradeDocumentFileWidget(forms.widgets.ClearableFileInput):
 
 
 class GradeDocumentForm(forms.ModelForm):
-    # see EvaluationForm (staff/forms.py) for details, why the following two fields are needed
     last_modified_time_2 = forms.DateTimeField(label=_("Last modified"), required=False, localize=True, disabled=True)
     last_modified_user_2 = forms.CharField(label=_("Last modified by"), required=False, disabled=True)
```

Comment:
should these also replaced with a LoggedModel in the long run?

Reply:
we could do that, but there's no real need. actually this information is probably never used at all.
i'm fine with both keeping it and changing it. what would you prefer?

Reply:
if it's not used lets not spend time on it.